internal/pkg/vm/actor/builtin/account/account.go

```diff
@@ -70,27 +67,27 @@ func (a *Actor) Method(id types.MethodID) (dispatch.Method, *dispatch.FunctionSi
 }
 
 // InitializeState for account actors does nothing.
-func (*Actor) InitializeState(storage runtime.LegacyStorage, initializerData interface{}) error {
-	state, ok := initializerData.(*State)
+func (*Actor) InitializeState(handle runtime.ActorStateHandle, initializerData interface{}) error {
```

Comment:
Question: is this code staying after we import new actor code from elsewhere?

Reply:
It's not relevant. 

The purpose of this got replaced by the constructor method on the spec. 
We still have the constructor wired to call this in our code since some tests and the gengen were calling this method directly. 

I haven't looked at the new actor code in the repo, so I can't tell you if the constructor method is building the state directly or calling another method to build the state. But it is now something internal to each actor and not needed by the VM.